drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/projects_importer.rb

```diff
@@ -66,11 +66,14 @@ def _run(hud_dir)
       sanity_check
       ProjectsImportAttempt.transaction do
         run_in_dir(hud_dir) do
-          upsert_funders
-          upsert_orgs
+          upsert_hud_file(file: 'Organization.csv', klass: GrdaWarehouse::Hud::Organization, hud_key: 'OrganizationID')
           upsert_projects
-          upsert_walkins
-          upsert_inventory
+          upsert_walkins # import flag indicating which projects accept walk-ins
+          upsert_hud_file(file: 'Funder.csv', klass: GrdaWarehouse::Hud::Funder, hud_key: 'FunderID')
+          upsert_hud_file(file: 'Inventory.csv', klass: GrdaWarehouse::Hud::Inventory, hud_key: 'InventoryID')
+          upsert_hud_file(file: 'ProjectCoC.csv', klass: GrdaWarehouse::Hud::ProjectCoc, hud_key: 'ProjectCoCID')
+          upsert_hud_file(file: 'HMISParticipation.csv', klass: GrdaWarehouse::Hud::HmisParticipation, hud_key: 'HMISParticipationID')
+          upsert_hud_file(file: 'CEParticipation.csv', klass: GrdaWarehouse::Hud::CeParticipation, hud_key: 'CEParticipationID')
```

Comment:
Seems like we could get the hud_key from klass.hud_key, but no harm in passing it in I guess.

Reply:
oh I didn't know that was a think on the GrdaWarehouse models too, I will update to use it